src/Fetch.res

```diff
@@ -82,6 +83,20 @@ module Headers = {
   external get: (t, string) => option<string> = "get"
   @send external has: (t, string) => bool = "has"
   @send external set: (t, string, string) => unit = "set"
+
+  /**
+    * Callback format is (. value, key)
+    */ @send
+  external forEach: (t, (. string, string) => unit) => unit = "forEach"
+
+  /**
+    * Callback format is (. value, key, headers)
+    */ @send
+  external forEachWithHeaders: (t, (. string, string, t) => unit) => unit = "forEach"
+
+  @send external entries: t => Js.Array2.array_like<(string, string)> = "entries"
+  @send external keys: t => Js.Array2.array_like<string> = "keys"
+  @send external values: t => Js.Array2.array_like<string> = "values"
```

Comment:
Could you use `Js.Array` instead of `Js.Array2` here? They're aliased to the same type, and I'm hoping `Array` will be removed and `Array2` renamed, so it should work out better in the long run I think.